components/base/sensorcontrolled/velocities.go

```diff
@@ -12,27 +13,58 @@ import (
 )
 
 // TODO: RSDK-5355 useControlLoop bool should be removed after testing.
-const useControlLoop = false
+const (
+	useControlLoop = false
+	rPiGain        = 0.00392157
+)
 
 // setupControlLoops uses the embedded config in this file to initialize a control
 // loop using the controls package and stor in on the sensor controlled base struct
 // the sensor base in the controllable interface that implements State and GetState
 // called by the endpoing logic of the control thread and the controlLoopConfig
 // is included at the end of this file.
-func setupControlLoops(sb *sensorBase) error {
-	// TODO: RSDK-5355 useControlLoop bool should be removed after testing
-	if useControlLoop {
-		loop, err := control.NewLoop(sb.logger, controlLoopConfig, sb)
-		if err != nil {
-			return err
-		}
-		sb.loop = loop
+func (sb *sensorBase) setupControlLoops() error {
+	// create control loop
+	loop, err := control.NewLoop(sb.logger, controlLoopConfig, sb)
+	if err != nil {
+		return err
+	}
+	if err := loop.Start(); err != nil {
+		return err
+	}
+	sb.loop = loop
 
-		if sb.loop != nil {
-			if err := sb.loop.Start(); err != nil {
-				return err
-			}
-		}
+	return nil
+}
+
+func (sb *sensorBase) updateControlConfig(
+	ctx context.Context, linearValue, angularValue float64,
+) error {
+	// set linear setpoint config
+	linConf := control.BlockConfig{
+		Name: "linear_setpoint",
+		Type: "constant",
+		Attribute: rdkutils.AttributeMap{
+			// conver mmPerSec to mPerSec
+			"constant_val": linearValue / 1000.0,
```

Comment:
I'd rather keep value because the PID itself doesn't care about units, and it fits the nomenclature of `GetSignalValue` and `SetSignalValue` which makes sense for a control loop.

Reply:
I'll move it and leave the names as is.